examples/next-openai-telemetry/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "next-openai-telemetry",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@ai-sdk/openai": "latest",
+    "@ai-sdk/react": "latest",
+    "@opentelemetry/api-logs": "0.52.1",
```

Comment:
Do you actually need these deps? I think only `@opentelemetry/api` needed, if anything at all.

Reply:
There were complaints about missing peer dependencies with pnpm (apparently not automatically installed), so I added this.